src/utils/parser/getSymbols.js

```diff
@@ -359,7 +371,8 @@ export function formatSymbols(source: Source) {
       ? symbol.parameterNames.join(", ")
       : "";
     const expression = symbol.expression || "";
-    return `${loc} ${exprLoc} ${expression} ${symbol.name} ${params}`;
+    const klass = symbol.klass || "";
+    return `${loc} ${exprLoc} ${expression} ${symbol.name} ${params} ${klass}`;
```

Comment:
hmm, looks like we're not including functions, which we probably should... perhaps we should update `formatSymbol` to map the entire dict of symbols:

```js
Object.keys(symbols).map(name => `${name}:\n ${symbols[name].map(summarize)}`
```